hackathon_site/event/jinja2/event/landing.html

```diff
@@ -27,292 +29,296 @@
     <li><a href="#faq">FAQ</a></li>
     <li><a href="#sponsors">Sponsors</a></li>
     <li><a href="#contact">Contact</a></li>
+    {% if localtime(now) > localtime(registration_open_date) %}
     <li>
-        <a class="btn-small waves-effect waves-light colorBtn">Login/Apply</a>
+        <a class="btn-small waves-effect waves-light coloredBtn">Login</a>
     </li>
+    {% endif %}
 </ul>
 
 <a href="#" data-target="nav-mobile" class="sidenav-trigger"><i class="material-icons">menu</i></a>
 {% endblock %}
 
 {% block body %}
-<div class="container">
-    <div class="section row" id="landing">
-        <div class="col container s12 m7">
-            <h1 class="row header">Hackathon Name</h1>
+<div class="ombreBackgroundLight">
+    <div class="section purpleLanding" id="landing">
+        <div class="container">
+            <br /><br />
+            <h1 class="header white-text">NewHacks</h1>
             <div class="row">
-                <h4 class="header s12">Subtitle</h4>
+                <h4 class="header s12 white-text longText">
+                    24 Hour Beginner Friendly Digital Hackathon
+                </h4>
             </div>
             <div class="row">
-                <h5>Location</h5>
-                <h5>
+                <h5 class="white-text">University of Toronto</h5>
+                <h5 class="white-text">
                     {# Update this logic if your event ends in a different month or year #}
-                    {{ localtime(event_start_date).strftime("%B %d") }}-{{ localtime(event_end_date).strftime("%d, %Y") }}
+                    {{ localtime(event_start_date).strftime("%B %-d") }}-{{ localtime(event_end_date).strftime("%-d, %Y") }}
                 </h5>
             </div>
+            {% if localtime(now) > localtime(registration_open_date) %}
             <div class="row">
-                <a class="btn-large waves-effect waves-light colorBtn">Login/Apply</a>
+                <a class="btn-large waves-effect waves-light secondaryColor coloredBtn">
+                    {% if localtime(now) > localtime(registration_close_date) %}
+                    Login
+                    {% else %}
+                    Login/Apply
```

Comment:
Just a note about this for the future: We'll have to do some logic when they log in, and either send them to the application page or dashboard depending whether they've filled out an application. Should be easy.